src/app.py

```diff
@@ -66,6 +67,22 @@ def with_plugins() -> Iterable[PluginManager]:
         plugins.cleanup()
 
 
+# See docs/users.md for documentation on the permission model.
+class UserRole(Enum):
+    # "role groups", used to grant a collection of "action roles"
+    nobody = auto()  # no permissions granted
```

Comment:
I'd be a bit worried about using `auto()` if we were storing the values somewhere as reordering the class fields could lead to a user accidentally becoming an admin.
But since as far as I can tell the enum instances are ephemeral it's fine 🤔

Reply:
I agree. I don't see a good alternative though (explicit integers are super-ugly, manual strings `nobody = "nobody"` are verbose, and I'm unsure about using StrEnum.

I've decided to just document this issue/caveat for now.